src/components/pages/PageResult.tsx

```diff
@@ -1,9 +1,40 @@
-import React, { memo, VFC } from 'react';
+import React, { memo, useContext, VFC } from 'react';
+import { InputDataContext } from '../../providers/InputDataProvider';
+import { LinkButton, SButtons } from '../atoms/button/LinkButton';
+
+import { SContainer } from '../organisms/Header';
 
 export const PageResult : VFC = memo(() => {
+
+  const {inputData}  = useContext(InputDataContext);
+
+  const {year , month , day} = inputData.birthday;
```

Comment:
スペースの空け方に誤りがあります。
以下のようにすると良いです。
```suggestion
  const { year, month, day } = inputData.birthday;
```

エディタの自動整形を使用すると良いかと思います。
https://qiita.com/mitashun/items/e2f118a9ca7b96b97840